grove/grovetccfg/grovetccfg.go

```diff
@@ -94,13 +99,13 @@ func CopyAndGzipFile(src, dst string) error {
 }
 
 // BackupFile copies the given file to a new file in the subdirectory "/remap_history", with the name suffixed by the current timestamp. Returns nil if the given file doesn't exist (nothing to back up).
-func BackupFile(path string) error {
+func BackupFile(path string, bkup_dir string) error {
```

Comment:
`bkup_dir` should be camel case, and while Go favors brevity `bkup` is a bit extreme, I'd recommend `backupDir`